ExpensesTracker/Forms/MainWindow.cs

```diff
@@ -139,27 +154,34 @@ private void SavePdfButton_Click(object sender, EventArgs e)
 
         private void RefreshAccountsData()
         {
+            accountBox.SelectedIndexChanged -= (SelectedAccountBox_SelectedIndexChanged);
             accountBox.DataSource = null;
             accountBox.DataSource = _accounts;
             accountBox.DisplayMember = "Name";
-            accountBox.SelectedItem = TargetAccount;
+            accountBox.SelectedIndexChanged += (SelectedAccountBox_SelectedIndexChanged);
+            accountBox.SelectedItem = _accounts.First();
         }
 
         public void ShowBalance()
```

Comment:
Make the next methods private:
SelectedAccountBox_SelectedIndexChanged
ActualizeTableRecords
ShowBalance

Reply:
Do methods private until otherwise is required.